legal-api/src/legal_api/resources/v2/business/business_filings/business_filings.py

```diff
@@ -1020,20 +1022,20 @@ def get_internal_filings(status=None):
         pending_filings = Filing.get_completed_filings_for_colin()
         for filing in pending_filings:
             filing_json = filing.filing_json
-            legal_entity = LegalEntity.find_by_internal_id(filing.legal_entity_id)
+            business = business_service.fetch_business_by_id(filing.legal_entity_id)
```

Comment:
I couldn't figure out a way to use `fetch_business` instead of `fetch_business_by_id` here. Ideally we don't need the `id` and can remove the implementation of `fetch_business_by_id`.

The tricky part is that we don't get a business identifier in the query, and the `Filing` model only stores either `legal_entity_id` and `alternate_name_id`.

Reply:
I think you should update this new function to something like `business_service.fetch_business_by_filing(filing)`.  And within the function, you can return `AlternateName` or `LegalEntity` using whichever field(`filing.legal_entity_id` or `filing.alternate_name_id`) is non-null in `filing`.